packages/modular-types/src/types.ts

```diff
@@ -0,0 +1,26 @@
+export type ModularTemplateType = 'app' | 'esm-view' | 'view' | 'package';
```

Comment:
If it's only types there, can't you just write a .d.ts file yourself, point to it in the package.json and don't have to build it? It will be not much different than what you have here, but with much less tooling complexity (yes, I'm advocating for import from source, but in this case there's no real build step)

Reply:
Good point, I hadn't thought of it this way, but I guess there is no reason *not* to, if the only thing that gets published under this package is types.

That said, is that the same as importing from source? I think we can still have this as a distinct package rather than a direct folder of types

Reply:
> That said, is that the same as importing from source? I think we can still have this as a distinct package rather than a direct folder of types

It should still be a workspace with its own package.json and the "types" field, only sans build step.